packages/data/src/onboarding/reducer.js

```diff
@@ -262,6 +264,51 @@ const onboarding = (
 				},
 				taskLists: getUpdatedTaskLists( state.taskLists, task ),
 			};
+		case TYPES.HIDE_TASK_LIST_ERROR:
+			return {
+				...state,
+				errors: {
+					...state.errors,
+					hideTaskList: error,
+				},
+				taskLists: state.taskLists.map( ( list ) => {
+					if ( taskListId === list.id ) {
+						return {
+							...list,
+							isHidden: false,
+						};
+					}
+					return list;
+				} ),
+			};
+		case TYPES.HIDE_TASK_LIST_REQUEST:
+			return {
+				...state,
+				requesting: {
+					...state.requesting,
+					undoSnoozeTask: true,
```

Comment:
I assume this should be `hideTaskList` and same goes for line 306?

Reply:
Good catch.  ©️ 🍝